misc/metrics/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-metrics"
 edition = "2021"
 rust-version = "1.65.0"
 description = "Metrics for libp2p"
-version = "0.13.0"
+version = "0.13.1"
```

Comment:
CI will tell you that this version needs to be aligned with what we have in `workspace.dependencies`.